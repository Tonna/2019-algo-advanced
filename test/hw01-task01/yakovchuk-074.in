15
1 1 1 2 1 1 2 1 7 7 8 4 7 2
6
5 9
10 1
12 9
1 3
14 4
8 3
