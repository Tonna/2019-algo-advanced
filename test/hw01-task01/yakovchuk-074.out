13
7
26
3
10
5
